src/components/Overview/Overview.react.js

```diff
@@ -142,72 +169,75 @@ class Overview extends Component{
                    Download the App to have access to SUSI on the go.</p>
                    <div className="description__text store_icons">
                      <Link className="playstore"
-                       target="_blank" to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"> Get It on Google Play
+                       target="_blank" to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"> Get It on Google Play"
```

Comment:
What is going on with the random `"`?

Reply:
Actually this line was showing as a pink strip in sublime, so some punctuation's end part was missing, so I completed it..

Reply:
Sorry, I got confused in Sublime's highlighting of this part of the code. Will remove the punctuation. Thanks for pointing it out. Sorry for the inconvenience.